newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -38,6 +38,53 @@ const writeJSONFile = (object: Object, filepath: string): Promise<void> => {
   }
 };
 
+const writeProjectFiles = ( project: gdProject, newFilepath: string): Promise<void> => {
```

Comment:
For this function, I was thinking of having only the common code with what there is already in the existing "saveProject" function :) 
https://github.com/4ian/GDevelop/pull/1162/files/452e887ff49defb9d0b77fc0625b80642521ba7b..49008995bc5ce7558eae0af668b701383f31163e#diff-6bbee279de98763ff959b65a37979ea7R95

So you would remove the call to `gd.ProjectResourcesCopier.copyAllResourcesTo ` and potentially other lines that are not in common, and put them back to `saveProjectAs`. The idea is to avoid the duplication of all the logic related to isProjectFolder and the project file writing. Pretty sure we don't need to duplicate this, it's a risk of introducing different behaviors later if we modify saveProject and not saveProjectAs or vice-versa.